### DISCUSSION TITLE
Client和Server的版本必须一致么？

### DISCUSSION THREAD

**Discussion by darknesstm** [apolloconfig/apollo#5156] [Q&A]
为了支持jdk17以上，必须使用2.0以上的客户端版本，这种情况下 服务端也必须升级么？

**Comment by nobodyiam** [comment]
一般建议是服务端的版本比客户端高，不过 apollo 的服务端接口比较稳定，所以不升级应该也可以，有一些新功能会不可用（例如 openapi 的一些新接口），具体可以看下服务端的 release note

**Reply by darknesstm** [reply]
最好有的兼容性列表，一般其实也不会升级client，但这次因为升级jdk必须升级client，但server是共用的，升级等操作担心有风险

**Reply by painAdmin** [reply]
同求，这最好有个列表，这样不要开发者每个release note 自己 排查是否符合要求，否则的风险太大，只能忍痛不升级了，

**Comment by painAdmin** [comment]
同问，如果服务端版本升级的话，客户端需要升级吗？不升级是否有问题，在线运行的服务如果，客户端全部匹配升级，会需要较大时间和风险，方案也需要制定？？？？？？？？？？？？？？？？？？？？？

**Reply by nobodyiam** [reply]
客户端版本比服务端低没问题的
